src/components/SuggestedAction/SuggestedAction.tsx

```diff
@@ -56,6 +57,7 @@ const SuggestedAction = ({
   viewWorkflow,
   disableAction,
   dismissAction,
+  viewMenuCallback,
```

Comment:
nit: rather than doing a check below we can define a default value of an empty function--so:
```
viewMenuCallback = () => {}
```